test/liquid_voting/delegations_test.exs

```diff
@@ -10,22 +10,36 @@ defmodule LiquidVoting.DelegationsTest do
       delegator = insert(:participant)
       delegate = insert(:participant)
       another_delegate = insert(:participant)
+      organization_id = Ecto.UUID.generate()
+      proposal_url = "https://www.someorg/proposalX"
 
       [
         valid_attrs: %{
           delegator_id: delegator.id,
           delegate_id: delegate.id,
-          organization_id: Ecto.UUID.generate()
+          organization_id: organization_id
         },
         update_attrs: %{
           delegator_id: delegator.id,
           delegate_id: another_delegate.id,
-          organization_id: Ecto.UUID.generate()
+          organization_id: organization_id
         },
         invalid_attrs: %{
           delegator_id: delegator.id,
           delegate_id: nil,
           organization_id: nil
+        },
+        valid_proposal_specific_attrs: %{
+          delegator_id: delegator.id,
+          delegate_id: delegate.id,
+          organization_id: organization_id,
+          proposal_url: proposal_url
+        },
+        update_proposal_specific_attrs: %{
```

Comment:
I don't see this used anywhere, looks like it's orphaned by latest changes

Reply:
Done - good eyes!